vortex-ffi/src/array.rs

```diff
@@ -42,19 +45,23 @@ pub unsafe extern "C" fn FFIArray_dtype(ffi_array: *const FFIArray) -> *const DT
 pub unsafe extern "C" fn FFIArray_get_field(
     ffi_array: *const FFIArray,
     index: u32,
+    error: *mut *mut FFIError,
 ) -> *const FFIArray {
-    let array = ffi_array.as_ref().vortex_expect("array null");
+    let result = (|| {
```

Comment:
i guess this is somewhere try-blocks would be useful huh

Reply:
Very, try2 plz